### DISCUSSION TITLE
Integration with Neorg presenter

### DISCUSSION THREAD

**Discussion by danymat** [Pocco81/TrueZen.nvim#71] [Show and tell]
Hello @Pocco81 and thank your for this wonderful plugin !

I currently wrote a module for Neorg called `presenter` which let you create slides from a `.norg` file.
I added the ability to add zen-mode support for both `TrueZen` and `zen-mode` plugins, which is why I wanted to share you, since we're using your plugin (users need the plugin to be installed via their plugin manager ofc).

This is the demo of what it is doing with TrueZen:

https://user-images.githubusercontent.com/5306901/142048075-1f44b5db-d768-4b77-a813-f65e8b2a9d40.mov

I hope that you like this integration, and of you have some setup options that feel like will be great to a presentation slide, feel free to share !

**Comment by pocco81** [comment]
I think it looks awesome! great job :+1:
